### PR TITLE
feat: generate SDKs for Looker 22.10

### PR DESCRIPTION
**Pull request by jkaster**

### CODE REVIEW

File: bin/sdk_gen [hunk 1]
```diff
@@ -172,16 +172,15 @@ const pullci = async (release) => {
...
-        // 'git add -A',
-        // `git commit -m "feat: generate SDKs for Looker ${release}"`,
-        /*
-        Test these out after first cut is merged to main
-        'git push'
-        create PR
+        'git add -A',
+        `git commit -m "feat: generate SDKs for Looker ${release}"`,
+        `git push origin ${branch}`,
+        /* create PR
```
Comment (josephaxisa) [inline bin/sdk_gen:183]: I believe we mentioned this will be removed in a separate PR so I won't let it block this.

**Review by josephaxisa — APPROVED** [review]
lgtm - thanks for your work on this John!

File: bin/sdk_gen (hunk 1 above)
Comment (jkaster) [reply bin/sdk_gen:183]: yeah, I'll get rid of the commented out PR notes for the next round. Thanks!
